src/panel/poi/ActionButtons.jsx

```diff
@@ -42,9 +47,17 @@ const ActionButtons = ({
 
     <Button
       className="poi_panel__action__favorite"
-      onClick={toggleStorePoi}
-      icon={isPoiInFavorite ? 'icon_star-filled' : 'star'}
+      data-active={isPoiInFavorite}
       title={_('Favorites', 'poi panel')}
+      onClick={toggleStorePoi}
+      style={{ borderColor: favoriteColor }}
+      icon={
+        <Heart
+          width={16}
+          color={favoriteColor}
+          fill={favoriteColor || 'white'}
```

Comment:
It doesn't change anything visually here, but technically the `fill` in this case should be `none` or `transparent`.

Reply:
![Peek 2020-12-10 15-42](https://user-images.githubusercontent.com/2981774/101786962-d5eb3d80-3afe-11eb-9ef5-049116c9a6fd.gif)
actually it does have a visual impact, thanks !